source/FreeRTOS_RA.c

```diff
@@ -264,7 +264,15 @@
         while( ( uxIndex + 1U ) < uxLast )
         {
             uint8_t ucType = pucBytes[ uxIndex ];
-            size_t uxLength = ( size_t ) pucBytes[ uxIndex + 1U ] * 8U;
+            size_t uxPrefixLength = ( size_t ) pucBytes[ uxIndex + 1U ];
+            size_t uxLength = uxPrefixLength * 8U;
```

Comment:
Why add uxPrefixLength here? If we just check the zero case, we can check uxLength directly.

Reply:
Yeah we can check that directly as well, but that does not justify what 0 check we are doing, i added it because the prefix length should not be zero ,  not the prefixlength*8